include/range/v3/view/map.hpp

```diff
@@ -124,6 +124,16 @@ namespace ranges
         /// \ingroup group-views
         RANGES_INLINE_VARIABLE(view<values_fn>, values)
     } // namespace view
+
+    namespace cpp20
+    {
+        namespace view
+        {
+            using ranges::view::keys;
+            using ranges::view::values;
+        }
+        // TODO(@cjdb): provide implementation for elements_view
```

Comment:
Shall I wait for this?

Reply:
I think it'd be better for me to make that a separate PR. I don't have the time to make that right now ☹️. Would you like me to remove the TODO?